docs/extending-typespec/emitters-basics.md

```diff
@@ -187,6 +185,37 @@ Instead, use the compiler [`host` interface](#todo) to access the file system. T
 
 In order to know where to emit files, the emitter context has a `emitterOutputDir` property that is automatically resolved using the `emitter-output-dir` built-in emitter options. This is set to `{cwd}/tsp-output/{emitter-name}` by default, but can be overridden by the user. Do not use the `compilerOptions.outputDir`
 
+## Handling scalars
+
+Scalars are types in TypeSpec that most likely have a primitive or built-in datastructure represneting those in the target language.
```

Comment:
```suggestion
Scalars are types in TypeSpec that most likely have a primitive or built-in datastructure representing those in the target language.
```

Reply:
hhm weird, speelcheck should have caught that

Reply:
hhm I guess it is 2 valid words combined